src/setupTests.js

```diff
@@ -12,3 +12,5 @@ const localStorageMock = {
 const uuid = { v4: jest.fn(() => 1) };
 global.uuid = uuid;
 global.localStorage = localStorageMock;
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 30000;
```

Comment:
Why are we setting a timeout here?

Reply:
The reason for the timeout is to give enough time for the async `login test` to pass instead of pushing to travis over again.

Reply:
Will it affect timeouts set?

Reply:
No, it won't it specific for the tests.